package.json

```diff
@@ -61,7 +61,8 @@
     "sweetalert2": "^6.6.6",
     "webrtc-adapter": "^4.2.1",
     "winston": "^2.3.1",
-    "zone.js": "^0.8.4"
+    "zone.js": "^0.8.4",
+    "email-verify": "0.2.1"
```

Comment:
I am sorry to comment on this after the PR has been merged, but it would have been a good idea to add this in the correct alphabetical order to make it easier to locate any package and maintain integrity.

Reply:
yes @paavininanda. Now that it is merged, best solution would be to create an Issue for it.